styles/base.css

```diff
@@ -78,43 +81,60 @@ h4,
 
 :root {
     --title-font-size: 1.75rem;
-    --button-text-font-size: 1.25rem;
 }
 
 @media (min-width: 992px) {
     :root {
         --title-font-size: 2rem;
-        --button-text-font-size: 1.5rem;
     }
 }
 
 .lg-link {
     font-size: var(--title-font-size);
 }
 
-.btn-link {
-    font-size: var(--button-text-font-size);
+/* Headlines */
+
+:root {
+    --h2-font-size: calc(32rem / 16);
+    --h2-font-weight: 500;
+    --h2-line-height: 120%;
+    --h2-font-size: calc(24rem / 16);
+    --h2-font-weight: 700;
+    --h2-line-height: 120%;
+}
+
+@media (min-width: 992px) {
+    :root {
+        --h2-font-size: calc(40rem / 16);
+        --h3-font-size: calc(32rem / 16);
+    }
 }
 
-/*
-Overwrite Bootstrap's heading font-sizes
-https://stackoverflow.com/questions/5410066/what-are-the-default-font-sizes-in-pixels-for-the-html-heading-tags-h1-h2
-*/
 
 h1 {
     font-size: 2rem;
     margin-top: 1.5rem;
     margin-bottom: 1.5rem;
 }
```

Comment:
Updating h1, h4 will happen in subsequent PR #37. It will be required for completing #66.